src/lib/users/dialogs/inst_user_first.rb

```diff
@@ -0,0 +1,661 @@
+        "Usernames have stricter restrictions than passwords. You can redefine the\n" \
+        "restrictions in the /etc/login.defs file. Read its man page for information.\n" \
+        "</p>\n"
+      ) +
+      _(
+        "<p>Check <b>Use this password for system administrator</b> if the " \
+        "same password as entered for the first user should be used for root.</p>"
+      ) +
+      _(
+        "<p>\nThe username and password created here are needed to log in " \
+        "and work with your Linux system. With <b>Automatic Login</b> enabled, " \
+        "the login procedure is skipped. This user is logged in automatically.</p>\n"
+      ) +
+      _(
+        "<p>\nHave mail for root forwarded to this user by checking <b>Receive System Mail</b>.</p>\n"
+      )
+
+      if import_available?
+        help += "<p>\n<b>" + _("Import User Data from a Previous Installation") + "</b>\n</p>\n"
+        help += _(
+          "<p>\nA previous Linux installation with local users has been detected.\n" \
+          "The information there can be used to create users in the system being installed.\n" \
+          "Use the <b>Choose Users</b> button to select some users. Their basic information will\n" \
+          "be imported.\n</p>\n"
+        )
+      end
+
+      help += "<p>\n<b>" + _("Skip User Creation") + "</b>\n</p>\n"
+      help += _(
+        "<p>\nSometimes root is the only needed local user, like in network environments\n" \
+        "with an authentication server. Select this option to proceed without creating\n" \
+        "a local user.\n</p>\n"
+      )
+
+      help
+    end
+
+  private
+
+    # Imports all the used YaST modules
+    #
+    # Importing them before class initialization has been observed to
+    # be problematic
+    def import_yast_modules
+      Yast.import "UI"
+      Yast.import "GetInstArgs"
+      Yast.import "Label"
+      Yast.import "Mode"
+      Yast.import "Popup"
+      Yast.import "ProductFeatures"
+      Yast.import "Progress"
+      Yast.import "Report"
+      Yast.import "UsersSimple"
+      Yast.import "Wizard"
+    end
+
+    # Initializes the instance variables used to configure a new user
+    #
+    # Requires @user to be set in advance
+    def init_user_attributes
+      @user_type = @user.fetch("type", "local")
+      @username = @user.fetch("uid", "")
+      @full_name = @user.fetch("cn", "")
+      @password = @user["userPassword"]
+
+      init_autologin
+      init_pw_for_root
+      init_root_mail
+    end
+
+    # Sets the initial default value for autologin
+    # Requires @user to be already set
+    def init_autologin
+      @autologin = UsersSimple.AutologinUsed
+      if @user.empty? && !@autologin
+        if ProductFeatures.GetBooleanFeature("globals", "enable_autologin") == true
+          @autologin = true
+        end
+        Builtins.y2debug("autologin default value: %1", @autologin)
```

Comment:
This is not the only `y2debug` in this file. There is still quite some old code, but... you know.